eng/tools/HelixTestRunner/ProcessUtil.cs

```diff
@@ -14,10 +14,10 @@
 
 namespace HelixTestRunner;
 
-public static class ProcessUtil
+public static partial class ProcessUtil
 {
-    [DllImport("libc", SetLastError = true, EntryPoint = "kill")]
-    private static extern int sys_kill(int pid, int sig);
+    [LibraryImportAttribute("libc", SetLastError = true, EntryPoint = "kill")]
```

Comment:
```suggestion
    [LibraryImport("libc", SetLastError = true, EntryPoint = "kill")]
```
The suffix `Attribute` is omitted usually?